### PR TITLE
Add dockerfile

### LINKED ISSUE #3: Add Dockerfile
Currently we don't have a dockerfile.
Having a docker file would help when experimenting in local Istio clusters for example.

wdyt? I can raise a PR if you think this will be help.

### PR DESCRIPTION
**Pull request by Pawan-Bishnoi**
Fixes : #3

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM golang
+WORKDIR /app
+COPY * ./
+RUN go build -o processorService
+EXPOSE 18080
+CMD ["./processorService"]
```
Comment (salrashid123) [inline Dockerfile]: suggest `CMD` -> `ENTRYPOINT` instead

**Review by salrashid123 — CHANGES_REQUESTED** [review]
looks good; thanks for the pr

File: Dockerfile (hunk 1 above)
Comment (Pawan-Bishnoi) [reply Dockerfile]: lost track of this PR. Updated. PTAL.

Any specific reason of using Entrypoint over Cmd?

Comment (salrashid123) [reply Dockerfile]: thanks,; no strong reason other than there aren't that many parameters users would modify while running the container

**Pull request merged**